frontend/controller/signupController.js

```diff
@@ -40,14 +40,26 @@ const verifyOTP = async (otp, payload, navigation,showAlert) => {
 };
 
 const BrandSignUp = async (payload, navigation,showAlert) => {
+  const data = new FormData();
+  data.append("name", payload.name);
+  data.append("email", payload.email);
+  data.append("password", payload.password);
```

Comment:
let's not show the password in plaintext here, either have it with eye icon or remove it

Reply:
Password gets encrypted in the backend. If we encrypt it in the frontend also then it will become hard to decrypt.

Reply:
We shouldn't be showing the password during preview 'coz user will unknowlingly expose the password if he creates the account infront of anyone no?

Reply:
Ok you mean in the preview we can able show the or hide the password. Is it? If the thing is that then wee need to implement a function show and hide password with eye icon in preview. I hope you get it.

Reply:
>If the thing is that then wee need to implement a function show and hide password with eye icon in preview. I hope you get it.


if we are gonna implement this atleast, i am good otherwise we should remove the password being shown in preview

Reply:
Ok I will add this one